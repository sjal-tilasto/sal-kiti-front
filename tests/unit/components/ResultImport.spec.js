import {
  createLocalVue,
  shallowMount,
  mount,
  RouterLinkStub
} from "@vue/test-utils";
import ResultImport from "@/components/ResultImport.vue";
import BootstrapVue from "bootstrap-vue";
import flushPromises from "flush-promises";

const localVue = createLocalVue();
localVue.use(BootstrapVue);

const $route = {
  params: { competition_id: 1 }
};

const resultTests = [
  {
    category: "SC",
    first_name: "Master",
    last_name: "Archer",
    organization: "LC",
    "kneel-1": 50,
    position: 1,
    position_pre: 2,
    result: 200,
    sport_id: 3456789
  },
  {
    category: "SC",
    organization: "LC",
    "kneel-1": "50",
    position: "1",
    position_pre: "2",
    result: "200",
    sport_id: "3456789"
  },
  {
    category: "SC",
    organization: "LC",
    result: 200,
    result_code: "DNF",
    sport_id: 3456789
  },
  {
    category: "SC",
    organization: "LC",
    result_code: "DNF",
    sport_id: 3456789
  },
  {
    category: "SC",
    organization: "LC",
    result_code: "Incorrect",
    sport_id: 3456789
  },
  {
    category: "SC",
    first_name: "Incorrect",
    last_name: "Incorrect",
    organization: "Village",
    result: 200,
    sport_id: 3456789
  },
  {
    category: "Village",
    organization: "LC",
    result: 200,
    sport_id: 1234567
  },
  {
    category: "Village",
    organization: "LC",
    position: "two",
    result: 200,
    sport_id: 1234567
  },
  {
    category: "Incorrect",
    organization: "Incorrect",
    result: 200,
    sport_id: 1234567
  },
  {
    category: "Incorrect",
    organization: "LC",
    result: 200,
    sport_id: 3456789
  },
  {
    category: "SC",
    organization: "LC",
    result: 200,
    sport_id: 3456789
  },
  {
    category: "SC",
    organization: "LC",
    "kneel-1": 150,
    result: 200,
    sport_id: 3456789
  },
  {
    category: "SC",
    organization: "LC",
    "incorrect-1": 50,
    result: 200,
    sport_id: 3456789
  },
  {
    first_name: "Master",
    last_name: "Archer",
    organization: "Incorrect",
    result: 500
  },
  {
    category: "SC",
    organization: "LC",
    position: 1,
    position_pre: 2,
    result: 200,
    sport_id: 3456789
  },
  {
    category: "SC",
    organization: "LC",
    "kneel-1": 50,
    result: 200,
    sport_id: 3456789
  },
  {
    category: "TeamC",
    organization: "LC",
    position_pre: 2,
    result: 200,
    sport_id: 3456789,
    team_name: "Team Failure 1"
  },
  {
    category: "TeamC",
    organization: "LC",
    result: 200,
    team_members: "1234567,3456789",
    team_name: "Team Success 1"
  },
  {
    category: "TeamC",
    organization: "LC",
    result: 190,
    sport_id_a: "1234567",
    sport_id_b: "3456789",
    first_name_b: "Ignored",
    last_name_b: "Archer",
    organization_b: "EXT",
    team_name: "Team Success 2"
  },
  {
    category: "TeamC",
    organization: "LC",
    result: 180,
    sport_id_a: "1234567",
    first_name_b: "External",
    last_name_b: "Archer",
    organization_b: "EXT",
    team_name: "Team Success 3"
  }
];

describe("ResultImport.vue", () => {
  it("renders result import form", async () => {
    const wrapper = shallowMount(ResultImport, {
      localVue,
      mocks: {
        $route
      },
      stubs: { "router-link": RouterLinkStub }
    });
    await flushPromises();
    expect(wrapper.element).toMatchSnapshot();
  });

  it("parses results", async () => {
    const wrapper = mount(ResultImport, {
      localVue,
      mocks: {
        $route
      },
      stubs: { "router-link": RouterLinkStub }
    });
    await flushPromises();
    wrapper.vm.$data.results = resultTests;
    wrapper.vm.parseResults();
    await flushPromises();
    expect(wrapper.element).toMatchSnapshot();
  });
});
