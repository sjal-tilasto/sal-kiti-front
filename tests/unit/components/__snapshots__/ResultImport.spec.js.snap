// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultImport.vue parses results 1`] = `
<div>
  <div
    class="row"
  >
    <div
      class="col"
    >
      <!---->
    </div>
  </div>
   
  <div
    class="row"
  >
    <div
      class="col"
    >
      <h4
        class="bg-sal-orange"
      >
        result.import
      </h4>
       
      <form
        class=""
      >
        <div
          class="form-group"
          id="input-group-file"
          role="group"
        >
          <!---->
          <div
            class="bv-no-focus-ring"
          >
            <div
              class="custom-file b-form-file is-invalid"
              id="__BVID__14__BV_file_outer_"
            >
              <input
                aria-required="true"
                class="custom-file-input is-invalid"
                id="__BVID__14"
                required="required"
                style="z-index: -5;"
                type="file"
              />
              <label
                class="custom-file-label"
                data-browse="Browse"
                for="__BVID__14"
              >
                <span
                  class="d-block form-file-text"
                  style="pointer-events: none;"
                >
                  import.file.select
                </span>
              </label>
            </div>
            <!---->
            <!---->
            <small
              class="form-text text-muted"
              id="input-group-file__BV_description_"
              tabindex="-1"
            >
              import.file.select_or_drop
            </small>
          </div>
        </div>
         
        <div
          class="form-group"
          id="input-group-type"
          role="group"
        >
          <!---->
          <div
            class="bv-no-focus-ring"
          >
            <div
              class="custom-control custom-radio"
            >
              <input
                class="custom-control-input"
                id="__BVID__16"
                name="filetype"
                type="radio"
                value="excel"
              />
              <label
                class="custom-control-label"
                for="__BVID__16"
              >
                Excel / CSV
              </label>
            </div>
             
            <div
              class="custom-control custom-radio"
            >
              <input
                class="custom-control-input"
                id="__BVID__17"
                name="filetype"
                type="radio"
                value="ianseo"
              />
              <label
                class="custom-control-label"
                for="__BVID__17"
              >
                Ianseo ASC
              </label>
            </div>
            <!---->
            <!---->
            <small
              class="form-text text-muted"
              id="input-group-type__BV_description_"
              tabindex="-1"
            >
              import.type
            </small>
          </div>
        </div>
         
        <!---->
         
        <div
          class="custom-control custom-checkbox"
        >
          <input
            class="custom-control-input"
            id="checkbox-dry-run"
            name="checkbox-dry-run"
            type="checkbox"
            value="true"
          />
          <label
            class="custom-control-label"
            for="checkbox-dry-run"
          >
            
          import.dry_run
        
          </label>
        </div>
         
        <button
          class="btn btn-orange space-right btn-light"
          type="submit"
        >
          submit
        </button>
      </form>
    </div>
  </div>
   
  <div
    class="row"
  >
    <div
      class="col"
    >
      <p>
        search.count: 19
      </p>
       
      <table
        aria-busy="false"
        aria-colcount="8"
        class="table b-table table-sm"
        id="__BVID__19"
        role="table"
      >
        <!---->
        <!---->
        <thead
          class=""
          role="rowgroup"
        >
          <!---->
          <tr
            class=""
            role="row"
          >
            <th
              aria-colindex="1"
              class=""
              role="columnheader"
              scope="col"
            >
              <div>
                import.status.label
              </div>
            </th>
            <th
              aria-colindex="2"
              class=""
              role="columnheader"
              scope="col"
            >
              <div>
                athlete.sport_id
              </div>
            </th>
            <th
              aria-colindex="3"
              class=""
              role="columnheader"
              scope="col"
            >
              <div>
                first_name
              </div>
            </th>
            <th
              aria-colindex="4"
              class=""
              role="columnheader"
              scope="col"
            >
              <div>
                last_name
              </div>
            </th>
            <th
              aria-colindex="5"
              class=""
              role="columnheader"
              scope="col"
            >
              <div>
                result.category
              </div>
            </th>
            <th
              aria-colindex="6"
              class=""
              role="columnheader"
              scope="col"
            >
              <div>
                result.result
              </div>
            </th>
            <th
              aria-colindex="7"
              class=""
              role="columnheader"
              scope="col"
            >
              <div>
                import.warning.label
              </div>
            </th>
            <th
              aria-colindex="8"
              class=""
              role="columnheader"
              scope="col"
            >
              <div>
                import.error.label
              </div>
            </th>
          </tr>
        </thead>
        <tbody
          role="rowgroup"
        >
          <!---->
          <tr
            class=""
            role="row"
          >
            <td
              aria-colindex="1"
              class=""
              role="cell"
            >
              import.status.success
            </td>
            <td
              aria-colindex="2"
              class=""
              role="cell"
            >
              3456789
            </td>
            <td
              aria-colindex="3"
              class=""
              role="cell"
            >
              Master
            </td>
            <td
              aria-colindex="4"
              class=""
              role="cell"
            >
              Archer
            </td>
            <td
              aria-colindex="5"
              class=""
              role="cell"
            >
              SC
            </td>
            <td
              aria-colindex="6"
              class=""
              role="cell"
            >
              <div>
                200
              </div>
            </td>
            <td
              aria-colindex="7"
              class=""
              role="cell"
            >
              <ul />
            </td>
            <td
              aria-colindex="8"
              class=""
              role="cell"
            >
              <ul />
            </td>
          </tr>
          <tr
            class=""
            role="row"
          >
            <td
              aria-colindex="1"
              class=""
              role="cell"
            >
              import.status.success
            </td>
            <td
              aria-colindex="2"
              class=""
              role="cell"
            >
              3456789
            </td>
            <td
              aria-colindex="3"
              class=""
              role="cell"
            >
              Master
            </td>
            <td
              aria-colindex="4"
              class=""
              role="cell"
            >
              Archer
            </td>
            <td
              aria-colindex="5"
              class=""
              role="cell"
            >
              SC
            </td>
            <td
              aria-colindex="6"
              class=""
              role="cell"
            >
              <div>
                
            DNF
            
                <div>
                  (200)
                </div>
              </div>
            </td>
            <td
              aria-colindex="7"
              class=""
              role="cell"
            >
              <ul />
            </td>
            <td
              aria-colindex="8"
              class=""
              role="cell"
            >
              <ul />
            </td>
          </tr>
          <tr
            class=""
            role="row"
          >
            <td
              aria-colindex="1"
              class=""
              role="cell"
            >
              import.status.success
            </td>
            <td
              aria-colindex="2"
              class=""
              role="cell"
            >
              3456789
            </td>
            <td
              aria-colindex="3"
              class=""
              role="cell"
            >
              Master
            </td>
            <td
              aria-colindex="4"
              class=""
              role="cell"
            >
              Archer
            </td>
            <td
              aria-colindex="5"
              class=""
              role="cell"
            >
              SC
            </td>
            <td
              aria-colindex="6"
              class=""
              role="cell"
            >
              <div>
                
            DNF
            
                <!---->
              </div>
            </td>
            <td
              aria-colindex="7"
              class=""
              role="cell"
            >
              <ul />
            </td>
            <td
              aria-colindex="8"
              class=""
              role="cell"
            >
              <ul />
            </td>
          </tr>
          <tr
            class=""
            role="row"
          >
            <td
              aria-colindex="1"
              class=""
              role="cell"
            >
              import.status.failure
            </td>
            <td
              aria-colindex="2"
              class=""
              role="cell"
            >
              3456789
            </td>
            <td
              aria-colindex="3"
              class=""
              role="cell"
            >
              Master
            </td>
            <td
              aria-colindex="4"
              class=""
              role="cell"
            >
              Archer
            </td>
            <td
              aria-colindex="5"
              class=""
              role="cell"
            >
              SC
            </td>
            <td
              aria-colindex="6"
              class=""
              role="cell"
            >
              <div>
                
            Incorrect
            
                <!---->
              </div>
            </td>
            <td
              aria-colindex="7"
              class=""
              role="cell"
            >
              <ul />
            </td>
            <td
              aria-colindex="8"
              class=""
              role="cell"
            >
              <ul>
                <li>
                  
              import.error.result_code
            
                </li>
              </ul>
            </td>
          </tr>
          <tr
            class=""
            role="row"
          >
            <td
              aria-colindex="1"
              class=""
              role="cell"
            >
              import.status.success
            </td>
            <td
              aria-colindex="2"
              class=""
              role="cell"
            >
              3456789
            </td>
            <td
              aria-colindex="3"
              class=""
              role="cell"
            >
              Master
            </td>
            <td
              aria-colindex="4"
              class=""
              role="cell"
            >
              Archer
            </td>
            <td
              aria-colindex="5"
              class=""
              role="cell"
            >
              SC
            </td>
            <td
              aria-colindex="6"
              class=""
              role="cell"
            >
              <div>
                200
              </div>
            </td>
            <td
              aria-colindex="7"
              class=""
              role="cell"
            >
              <ul>
                <li>
                  
              import.warning.first_name
            
                </li>
                <li>
                  
              import.warning.last_name
            
                </li>
              </ul>
            </td>
            <td
              aria-colindex="8"
              class=""
              role="cell"
            >
              <ul />
            </td>
          </tr>
          <tr
            class=""
            role="row"
          >
            <td
              aria-colindex="1"
              class=""
              role="cell"
            >
              import.status.failure
            </td>
            <td
              aria-colindex="2"
              class=""
              role="cell"
            >
              1234567
            </td>
            <td
              aria-colindex="3"
              class=""
              role="cell"
            />
            <td
              aria-colindex="4"
              class=""
              role="cell"
            />
            <td
              aria-colindex="5"
              class=""
              role="cell"
            >
              Village
            </td>
            <td
              aria-colindex="6"
              class=""
              role="cell"
            >
              <div>
                200
              </div>
            </td>
            <td
              aria-colindex="7"
              class=""
              role="cell"
            >
              <ul />
            </td>
            <td
              aria-colindex="8"
              class=""
              role="cell"
            >
              <ul>
                <li>
                  
              import.error.category_incorrect
            
                </li>
              </ul>
            </td>
          </tr>
          <tr
            class=""
            role="row"
          >
            <td
              aria-colindex="1"
              class=""
              role="cell"
            >
              import.status.failure
            </td>
            <td
              aria-colindex="2"
              class=""
              role="cell"
            >
              1234567
            </td>
            <td
              aria-colindex="3"
              class=""
              role="cell"
            />
            <td
              aria-colindex="4"
              class=""
              role="cell"
            />
            <td
              aria-colindex="5"
              class=""
              role="cell"
            >
              Village
            </td>
            <td
              aria-colindex="6"
              class=""
              role="cell"
            >
              <div>
                200
              </div>
            </td>
            <td
              aria-colindex="7"
              class=""
              role="cell"
            >
              <ul />
            </td>
            <td
              aria-colindex="8"
              class=""
              role="cell"
            >
              <ul>
                <li>
                  
              import.error.category_incorrect
            
                </li>
              </ul>
            </td>
          </tr>
          <tr
            class=""
            role="row"
          >
            <td
              aria-colindex="1"
              class=""
              role="cell"
            >
              import.status.failure
            </td>
            <td
              aria-colindex="2"
              class=""
              role="cell"
            >
              1234567
            </td>
            <td
              aria-colindex="3"
              class=""
              role="cell"
            />
            <td
              aria-colindex="4"
              class=""
              role="cell"
            />
            <td
              aria-colindex="5"
              class=""
              role="cell"
            >
              Incorrect
            </td>
            <td
              aria-colindex="6"
              class=""
              role="cell"
            >
              <div>
                200
              </div>
            </td>
            <td
              aria-colindex="7"
              class=""
              role="cell"
            >
              <ul />
            </td>
            <td
              aria-colindex="8"
              class=""
              role="cell"
            >
              <ul>
                <li>
                  
              import.error.category_incorrect
            
                </li>
              </ul>
            </td>
          </tr>
          <tr
            class=""
            role="row"
          >
            <td
              aria-colindex="1"
              class=""
              role="cell"
            >
              import.status.failure
            </td>
            <td
              aria-colindex="2"
              class=""
              role="cell"
            >
              3456789
            </td>
            <td
              aria-colindex="3"
              class=""
              role="cell"
            />
            <td
              aria-colindex="4"
              class=""
              role="cell"
            />
            <td
              aria-colindex="5"
              class=""
              role="cell"
            >
              Incorrect
            </td>
            <td
              aria-colindex="6"
              class=""
              role="cell"
            >
              <div>
                200
              </div>
            </td>
            <td
              aria-colindex="7"
              class=""
              role="cell"
            >
              <ul />
            </td>
            <td
              aria-colindex="8"
              class=""
              role="cell"
            >
              <ul>
                <li>
                  
              import.error.category_incorrect
            
                </li>
              </ul>
            </td>
          </tr>
          <tr
            class=""
            role="row"
          >
            <td
              aria-colindex="1"
              class=""
              role="cell"
            >
              import.status.success
            </td>
            <td
              aria-colindex="2"
              class=""
              role="cell"
            >
              3456789
            </td>
            <td
              aria-colindex="3"
              class=""
              role="cell"
            >
              Master
            </td>
            <td
              aria-colindex="4"
              class=""
              role="cell"
            >
              Archer
            </td>
            <td
              aria-colindex="5"
              class=""
              role="cell"
            >
              SC
            </td>
            <td
              aria-colindex="6"
              class=""
              role="cell"
            >
              <div>
                200
              </div>
            </td>
            <td
              aria-colindex="7"
              class=""
              role="cell"
            >
              <ul />
            </td>
            <td
              aria-colindex="8"
              class=""
              role="cell"
            >
              <ul />
            </td>
          </tr>
          <tr
            class=""
            role="row"
          >
            <td
              aria-colindex="1"
              class=""
              role="cell"
            >
              import.status.failure
            </td>
            <td
              aria-colindex="2"
              class=""
              role="cell"
            >
              3456789
            </td>
            <td
              aria-colindex="3"
              class=""
              role="cell"
            >
              Master
            </td>
            <td
              aria-colindex="4"
              class=""
              role="cell"
            >
              Archer
            </td>
            <td
              aria-colindex="5"
              class=""
              role="cell"
            >
              SC
            </td>
            <td
              aria-colindex="6"
              class=""
              role="cell"
            >
              <div>
                200
              </div>
            </td>
            <td
              aria-colindex="7"
              class=""
              role="cell"
            >
              <ul />
            </td>
            <td
              aria-colindex="8"
              class=""
              role="cell"
            >
              <ul>
                <li>
                  
              import.error.result_partial_max
            
                </li>
              </ul>
            </td>
          </tr>
          <tr
            class=""
            role="row"
          >
            <td
              aria-colindex="1"
              class=""
              role="cell"
            >
              import.status.success
            </td>
            <td
              aria-colindex="2"
              class=""
              role="cell"
            >
              3456789
            </td>
            <td
              aria-colindex="3"
              class=""
              role="cell"
            >
              Master
            </td>
            <td
              aria-colindex="4"
              class=""
              role="cell"
            >
              Archer
            </td>
            <td
              aria-colindex="5"
              class=""
              role="cell"
            >
              SC
            </td>
            <td
              aria-colindex="6"
              class=""
              role="cell"
            >
              <div>
                200
              </div>
            </td>
            <td
              aria-colindex="7"
              class=""
              role="cell"
            >
              <ul />
            </td>
            <td
              aria-colindex="8"
              class=""
              role="cell"
            >
              <ul />
            </td>
          </tr>
          <tr
            class=""
            role="row"
          >
            <td
              aria-colindex="1"
              class=""
              role="cell"
            >
              import.status.failure
            </td>
            <td
              aria-colindex="2"
              class=""
              role="cell"
            />
            <td
              aria-colindex="3"
              class=""
              role="cell"
            >
              Master
            </td>
            <td
              aria-colindex="4"
              class=""
              role="cell"
            >
              Archer
            </td>
            <td
              aria-colindex="5"
              class=""
              role="cell"
            />
            <td
              aria-colindex="6"
              class=""
              role="cell"
            >
              <div>
                500
              </div>
            </td>
            <td
              aria-colindex="7"
              class=""
              role="cell"
            >
              <ul />
            </td>
            <td
              aria-colindex="8"
              class=""
              role="cell"
            >
              <ul>
                <li>
                  
              import.error.category_missing
            
                </li>
              </ul>
            </td>
          </tr>
          <tr
            class=""
            role="row"
          >
            <td
              aria-colindex="1"
              class=""
              role="cell"
            >
              import.status.success
            </td>
            <td
              aria-colindex="2"
              class=""
              role="cell"
            >
              3456789
            </td>
            <td
              aria-colindex="3"
              class=""
              role="cell"
            >
              Master
            </td>
            <td
              aria-colindex="4"
              class=""
              role="cell"
            >
              Archer
            </td>
            <td
              aria-colindex="5"
              class=""
              role="cell"
            >
              SC
            </td>
            <td
              aria-colindex="6"
              class=""
              role="cell"
            >
              <div>
                200
              </div>
            </td>
            <td
              aria-colindex="7"
              class=""
              role="cell"
            >
              <ul />
            </td>
            <td
              aria-colindex="8"
              class=""
              role="cell"
            >
              <ul />
            </td>
          </tr>
          <tr
            class=""
            role="row"
          >
            <td
              aria-colindex="1"
              class=""
              role="cell"
            >
              import.status.success
            </td>
            <td
              aria-colindex="2"
              class=""
              role="cell"
            >
              3456789
            </td>
            <td
              aria-colindex="3"
              class=""
              role="cell"
            >
              Master
            </td>
            <td
              aria-colindex="4"
              class=""
              role="cell"
            >
              Archer
            </td>
            <td
              aria-colindex="5"
              class=""
              role="cell"
            >
              SC
            </td>
            <td
              aria-colindex="6"
              class=""
              role="cell"
            >
              <div>
                200
              </div>
            </td>
            <td
              aria-colindex="7"
              class=""
              role="cell"
            >
              <ul />
            </td>
            <td
              aria-colindex="8"
              class=""
              role="cell"
            >
              <ul />
            </td>
          </tr>
          <tr
            class=""
            role="row"
          >
            <td
              aria-colindex="1"
              class=""
              role="cell"
            >
              import.status.failure
            </td>
            <td
              aria-colindex="2"
              class=""
              role="cell"
            >
              3456789
            </td>
            <td
              aria-colindex="3"
              class=""
              role="cell"
            />
            <td
              aria-colindex="4"
              class=""
              role="cell"
            />
            <td
              aria-colindex="5"
              class=""
              role="cell"
            >
              TeamC
            </td>
            <td
              aria-colindex="6"
              class=""
              role="cell"
            >
              <div>
                200
              </div>
            </td>
            <td
              aria-colindex="7"
              class=""
              role="cell"
            >
              <ul />
            </td>
            <td
              aria-colindex="8"
              class=""
              role="cell"
            >
              <ul>
                <li>
                  
              import.error.category_team
            
                </li>
              </ul>
            </td>
          </tr>
          <tr
            class=""
            role="row"
          >
            <td
              aria-colindex="1"
              class=""
              role="cell"
            >
              import.status.success
            </td>
            <td
              aria-colindex="2"
              class=""
              role="cell"
            />
            <td
              aria-colindex="3"
              class=""
              role="cell"
            />
            <td
              aria-colindex="4"
              class=""
              role="cell"
            />
            <td
              aria-colindex="5"
              class=""
              role="cell"
            >
              TeamC
            </td>
            <td
              aria-colindex="6"
              class=""
              role="cell"
            >
              <div>
                200
              </div>
            </td>
            <td
              aria-colindex="7"
              class=""
              role="cell"
            >
              <ul />
            </td>
            <td
              aria-colindex="8"
              class=""
              role="cell"
            >
              <ul />
            </td>
          </tr>
          <tr
            class=""
            role="row"
          >
            <td
              aria-colindex="1"
              class=""
              role="cell"
            >
              import.status.success
            </td>
            <td
              aria-colindex="2"
              class=""
              role="cell"
            />
            <td
              aria-colindex="3"
              class=""
              role="cell"
            />
            <td
              aria-colindex="4"
              class=""
              role="cell"
            />
            <td
              aria-colindex="5"
              class=""
              role="cell"
            >
              TeamC
            </td>
            <td
              aria-colindex="6"
              class=""
              role="cell"
            >
              <div>
                190
              </div>
            </td>
            <td
              aria-colindex="7"
              class=""
              role="cell"
            >
              <ul />
            </td>
            <td
              aria-colindex="8"
              class=""
              role="cell"
            >
              <ul />
            </td>
          </tr>
          <tr
            class=""
            role="row"
          >
            <td
              aria-colindex="1"
              class=""
              role="cell"
            >
              import.status.success
            </td>
            <td
              aria-colindex="2"
              class=""
              role="cell"
            />
            <td
              aria-colindex="3"
              class=""
              role="cell"
            />
            <td
              aria-colindex="4"
              class=""
              role="cell"
            />
            <td
              aria-colindex="5"
              class=""
              role="cell"
            >
              TeamC
            </td>
            <td
              aria-colindex="6"
              class=""
              role="cell"
            >
              <div>
                180
              </div>
            </td>
            <td
              aria-colindex="7"
              class=""
              role="cell"
            >
              <ul />
            </td>
            <td
              aria-colindex="8"
              class=""
              role="cell"
            >
              <ul />
            </td>
          </tr>
          <!---->
          <!---->
        </tbody>
        <!---->
      </table>
    </div>
  </div>
   
  <div
    class="row"
  >
    <div
      class="col"
    >
      <a
        class="btn btn-orange btn-light"
        href="#"
        target="_self"
      >
        import.show_results
      
      </a>
       
      <button
        class="btn space-right float-right btn-outline-info"
        type="button"
      >
        
        import.debug_excel
      
      </button>
    </div>
  </div>
</div>
`;

exports[`ResultImport.vue renders result import form 1`] = `
<div>
  <b-row-stub
    tag="div"
  >
    <b-col-stub
      tag="div"
    >
      <!---->
    </b-col-stub>
  </b-row-stub>
   
  <b-row-stub
    tag="div"
  >
    <b-col-stub
      tag="div"
    >
      <h4
        class="bg-sal-orange"
      >
        result.import
      </h4>
       
      <b-form-stub>
        <b-form-group-stub
          description="import.file.select_or_drop"
          feedbackarialive="assertive"
          id="input-group-file"
          labelfor="input-file"
        >
          <b-form-file-stub
            accept=""
            browsetext="Browse"
            dropplaceholder="import.file.drop"
            filenameformatter="function _default() {
        return getComponentConfig(componentKey, prop, isFunction(defaultValue) ? defaultValue() : defaultValue);
      }"
            nodropplaceholder="Not allowed"
            placeholder="import.file.select"
            required="true"
          />
        </b-form-group-stub>
         
        <b-form-group-stub
          description="import.type"
          feedbackarialive="assertive"
          id="input-group-type"
          labelfor="input-type"
        >
          <b-form-radio-stub
            checked="excel"
            name="filetype"
            value="excel"
          >
            Excel / CSV
          </b-form-radio-stub>
           
          <b-form-radio-stub
            checked="excel"
            name="filetype"
            value="ianseo"
          >
            Ianseo ASC
          </b-form-radio-stub>
        </b-form-group-stub>
         
        <!---->
         
        <b-form-checkbox-stub
          id="checkbox-dry-run"
          name="checkbox-dry-run"
          value="true"
        >
          
          import.dry_run
        
        </b-form-checkbox-stub>
         
        <b-button-stub
          class="btn-orange space-right"
          tag="button"
          target="_self"
          type="submit"
          variant="light"
        >
          submit
        </b-button-stub>
      </b-form-stub>
    </b-col-stub>
  </b-row-stub>
   
  <!---->
   
  <b-row-stub
    tag="div"
  >
    <b-col-stub
      tag="div"
    >
      <!---->
       
      <!---->
    </b-col-stub>
  </b-row-stub>
</div>
`;
